packages/ripple-nuxt-tide/modules/landing-page/tide.config.js

```diff
@@ -460,6 +460,15 @@ module.exports = {
         props: {
           'formLink': ['field_form_link', 'uri']
         }
+      },
+
+      'paragraph--data_table': {
+        component: 'rpl-data-table',
+        props: {
+          caption: ['field_data_table_content', 'caption'],
+          isRowOriented: ['field_row_oriented_table'],
+          items: ['field_data_table_content', 'value']
```

Comment:
@rubyllamadora When we design the props for Vue component, we should think without the backend data structure first. Then we can use mapping filter to transform the Tide data into Vue prop data structure. So we keep them decoupled and more easy to use for non-Tide users.

Reply:
Totally agree about setting the component prop structure first. Ugh thanks Tim. I'll use mapping filter to transform data so it's compatible with the component.

Reply:
Updated this to convert Object into nested array.